templates/managed/kafka.html

```diff
@@ -140,6 +140,45 @@ <h3>
   </div>
 </section>
 
+<section class="p-strip--suru-accent is-dark" id="webinar">
+  <div class="row">
+    <div class="col-4 u-hide--small">
+      <div class="p-card">
+        <a href="https://www.brighttalk.com/webcast/6793/439718" aria-hidden="true" tabindex="-1">
+          {{
+            image(
+              url="https://assets.ubuntu.com/v1/e40b01bd-kafka-in-production-webinar.png",
+              alt="",
+              width="640",
+              height="360",
+              hi_def=True,
+            ) | safe
+          }}
+        </a>
+      </div>
+    </div>
+    <div class="col-8 col-start-large-6 u-vertically-center">
+      <div class="p-heading-icon--small">
+        <div class="p-heading-icon__header">
+          {{
+            image(
+              url="https://assets.ubuntu.com/v1/816ae23b-Webinar+-+white.svg",
+              alt="",
+              width="32",
+              height="32",
+              hi_def=True,
+              attrs={"class": "p-heading-icon__img"},
+            ) | safe
+          }}
+          <p class="p-muted-heading u-sv3" style="color: #fff; line-height: 1.5rem;">WEBINAR</p>
+        </div>
+        <h4>Kafka can be challenging. Learn how we simplify it</h4>
```

Comment:
There is a full stop at the end here in the copy doc
```suggestion
        <h4>Kafka can be challenging. Learn how we simplify it.</h4>
```

Reply:
I took that off as its a title but I guess if its in the copy it should be there